client-v2/src/lib/dnd/__tests__/dnd.spec.tsx

```diff
@@ -57,7 +57,7 @@ const runDrag = (type: any, data?: any, json: boolean = true) => (
 const setup = (jsx: React.ReactElement<any>) =>
   TestRenderer.create(jsx).getInstance();
 
-describe('Guration', () => {
+describe('Curation', () => {
```

Comment:
🤣 this isn't a typo, although @RichieAHB might be happy to see this change -- Guration was the WIP name of the drag and drop library! I don't think there are any other references to it in the codebase now (other than a dependency in package.json we can probably remore) so this is fine 😄

Reply:
🙈 wow. Oh okay!